src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends Figure {
+    public static final int FIGURE_COUNT = 5;
+    private static final Random random = new Random();
+    private static final Color WHITE = Color.WHITE;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(FIGURE_COUNT);
+        String color = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(DEFAULT_LENGTH) + 1;
```

Comment:
It's confusing why we pass Default Length here. Create a separate constant (also you won't need + 1 then)

Reply:
Still not fixed. Is unclear wehy exactly DEfault length is in nextint, You better clarify by creating new constant with proper meaning